### ISSUE TITLE
support for regtest 

### ISSUE THREAD

**Issue by 0xae** [etotheipi/BitcoinArmory#348]
Does armory support regtest ?

**Comment by droark** [comment]
Yes (AFAIK)...[under the new repo](https://github.com/goatpig/BitcoinArmory) (and [website](https://btcarmory.com/)). This repo has been dead for years. Please close this issue.
